airbyte-integrations/connectors/source-asana/integration_tests/abnormal_state.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
if you don't use incremental tests then you can drop this file

Reply:
Done.